tests/storage/test_delete.py

```diff
@@ -0,0 +1,79 @@
+import hashlib
+import os
+import glob
+import traceback
+import re
+import binascii
+import logging
+import sys
+sys.path.append("..")
+import time
+from logger import Logger, LOG_FOLDER, LOG_EXTENSION
+import argparse
+
+
+extension = ".txt"
+
+def get_args():
+    parser = argparse.ArgumentParser(description='Arguments', formatter_class = argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument( "--file_name", dest="file_name",
+                        required = True,
+                        help='Name of the file')
+    parser.add_argument("--dest", dest="path",
+                        required=True,
+                        help='Specify the path')
+    args = parser.parse_args()
+    return args
+
+
+class Delete():
```

Comment:
Can't we have a parent class, inherited from all the tests? Then I would set the parameters in a specific dictionary, implement the logging logic on the constructor and even the logic of "log_params" (that would list all parameters in the dictionary mentioned before).